### PR TITLE
Fixes for unstable simulation

### PR DESCRIPTION
**Pull request by mregen**
- Update nuget packages in simulation to follow security recommendations. Move to .Net Core 2.0
- When discovery is enabled too many messages are sent to TSI and cause throttling. Use 2 TSI units.
- Make MES startup code more robust and implement the reconnect handler.
- Fix a bug in CreateCerts, which creates the cert in local time zone, not in Utc, which may cause deployment to fail.

### CODE REVIEW

File: Simulation/Factory/Station/Station.csproj [hunk 1]
```diff
@@ -1,17 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```
Comment (marcschier) [inline Simulation/Factory/Station/Station.csproj:4]: 2.0 is out of support in a month or so, let's go with 2.2.

**Review by marcschier — APPROVED** [review]
We should go with 2.1 or 2.2 for dotnet runtime.

**Comment by marcschier** [comment]
Nice work!!!

**Review by marcschier — APPROVED** [review]

File: Simulation/Factory/Station/Station.csproj (hunk 1 above)
Comment (mregen) [reply Simulation/Factory/Station/Station.csproj:4]: its just the api level, no need here to bump it up...

**Pull request merged**